crates/sui-indexer-alt-graphql/src/api/mutation.rs

```diff
@@ -12,18 +31,60 @@ pub struct Mutation;
 impl Mutation {
     /// Execute a transaction, committing its effects on chain.
     ///
-    /// - `transaction` contains the transaction data in the desired format.
+    /// - `transaction_data_bcs` contains the BCS-encoded transaction data (Base64-encoded).
```

Comment:
It's a bit confusing, but in our doc comments, we should use camel case to refer to field names, because `async-graphql` automatically converts their names like that.

```suggestion
    /// - `transactionDataBcs` contains the BCS-encoded transaction data (Base64-encoded).
```